ReactiveCocoa/Swift/Atomic.swift

```diff
@@ -74,3 +121,47 @@ public final class Atomic<Value> {
 		return try action(_value)
 	}
 }
+
+public struct AnyAtomic<Value> {
```

Comment:
I'm not happy with this name )). This thing is basically a wrapper around `AnyAtomicState`, hiding its details, i.e. separate `withValue` logic for constant and changeable things.